app/serializers/user.js

```diff
@@ -5,7 +5,11 @@ export default ApplicationSerializer.extend({
   serialize(snapshot, options) {
     const json = this._super(...arguments);
     if (options && options.includeId) {
-      json.data.attributes = pick(json.data.attributes, ['email', 'password']);
+      if (snapshot.adapterOptions && snapshot.adapterOptions.updateMode === 'profile') {
```

Comment:
@niranjan94 The model contains attributes which will be in the payload but are not expected by the server. eg. isAdmin, createdAt..

So should I update the model and add readonly for them or omit them while serializing?